### ISSUE TITLE
Debugger listener starting error issue 

### ISSUE THREAD

**Issue by akshay-joshi** [akshay-joshi/TestRMToHub#643]
**This issue has been migrated from Redmine.**

Author Name: **Vishal Sawale** (@vishalsawale9)
Original Redmine Issue: [1376](https://redmine.postgresql.org/issues/1376)

Assignee: Vishal Sawale

---
We are getting an error in debugger while debugging the function on web based version pgadmin 4.
 
1.Launch pgAdmin4:
2. Perform pre-req verification for debugger
3. create function by query tool.
--------------------
CREATE OR REPLACE FUNCTION public.test123()
RETURNS character AS
$BODY$
begin
return 1;
end;
$BODY$
LANGUAGE plpgsql VOLATILE
COST 100;

4. Expand PG-9.5 server
5. Right click on the newly created function and click debug.
6. Check the result. Error has been showed. Screenshot attached.

Expected result: debugger should start 

OS - windows server 2012
Browser - IE 10, chrome - 51.0.2704.84 , firefox 47.0
commit ID- c702ef9c4864afa40056ec85983774a948a8bfab

---

- [Screen Shot 2016-06-17 at 2.48.52 pm.png](https://redmine.postgresql.org/attachments/download/1330/Screen%20Shot%202016-06-17%20at%202.48.52%20pm.png) (Vishal Sawale)
- [pgadmin4_1376.log](https://redmine.postgresql.org/attachments/download/1491/pgadmin4_1376.log) (Vishal Sawale)

**Comment by akshay-joshi** [comment]
Author Name: **Vishal Sawale** (@vishalsawale9)



---

-  fixed_version_id  was configured    as 32

**Comment by akshay-joshi** [comment]
Author Name: **Dave Page** (@dpage)



---

-  fixed_version_id  was changed from   32   to 33

**Comment by akshay-joshi** [comment]
Author Name: **Dave Page** (@dpage)


---
Can you confirm this is still an issue please? IF so, please provide debug level logs from immediately before until immediately after the error.

---

-  status_id  was changed from    New   to  In Testing 
-  assigned_to_id  was configured    as 316 
-  fixed_version_id  was changed from   33   to 34

**Comment by akshay-joshi** [comment]
Author Name: **Vishal Sawale** (@vishalsawale9)


---
Attached are the logs of pgadmin debug levels from immediately before until immediately after the error.

---

-  1491  was configured    as pgadmin4_1376.log 
-  status_id  was changed from    In Testing   to  New 

---

- [pgadmin4_1376.log](https://redmine.postgresql.org/attachments/download/1491/pgadmin4_1376.log) (Vishal Sawale)

**Comment by akshay-joshi** [comment]
Author Name: **Dave Page** (@dpage)



---

-  fixed_version_id  was changed from   34   to 36

**Comment by akshay-joshi** [comment]
Author Name: **Neel Patel** (@neel5481)


---

This is not an issue of debugger in pgAdmin4 but issue with the daily build installers 9.5 where plugin_debugger.dll was not loaded correctly.
We have also checked this issue with terminal instead of using pgAdmin4 and we are also getting the same result.

After reinstalling the release version of PG9.5 installer, everything works fine.


---

-  status_id  was changed from    New   to  In Progress
